src/states.jl

```diff
@@ -491,6 +493,88 @@ function PhaseEquil_ρpq(
     return PhaseEquil{FT, typeof(param_set)}(param_set, ρ, e_int, q_tot, T)
 end
 
+
+#####
+##### Anelastic-equilibrium states
+#####
+"""
+    AnelasticPhaseEquil{FT} <: AbstractPhaseEquil{FT}
+
+A thermodynamic state assuming thermodynamic equilibrium (therefore, saturation adjustment
+may be needed).
+
+# Constructors
+
+    PhaseEquil(param_set, ρ, p, e_int, q_tot, T)
+
+# Fields
+
+$(DocStringExtensions.FIELDS)
+"""
+
+struct AnelasticPhaseEquil{FT, PS} <: AbstractPhaseEquil{FT}
+    "parameter set, used to dispatch planet parameter function calls"
+    param_set::PS
+    "density of air (potentially moist)"
```

Comment:
maybe just to be extra clear we could write here that it's not the reference density?